### ISSUE TITLE
Add rake db2local to etmoses

### ISSUE THREAD

**Issue by grdw** [quintel/etmoses#907]
I'm working from home currently and I forgot the adapter to my Mac. No worrries, since there are 2 other laptops in the building where I can work from. Only downside is that there's no wonderfull program called Sequelpro for Linux. There are alternatives but most of them don't work/aren't developed for since 2005/etc. etc. 

So my idea is to include rake db2local just as etmodel, etengine. So I don't have to create a database export on a Mac, put it on Dropbox and than import it from my Linux machine.

**Comment by antw** [comment]
Agreed. Would be nice if it also downloaded a copy of the profiles otherwise the DB might refer to profiles which don't exist on your local machine.

**Comment by grdw** [comment]
> Would be nice if it also downloaded a copy of the profiles otherwise the DB might refer to profiles which don't exist on your local machine.

I already have a tiny bash script in my etmoses folder which does this. :sweat_smile: 

````bash
#!/bin/bash

rsync -avzr --delete ubuntu@beta.moses.et-model.com:/home/ubuntu/etmoses/current/public/system/ public/system
````
